spec/dynamoid/persistence_spec.rb

```diff
@@ -2293,6 +2293,29 @@ def around_update_callback
       end.not_to change { obj.class.count }
     end
 
+    context 'disable_create_table_on_save' do
+
+      before do
+        Dynamoid.configure do |config|
+          config.create_table_on_save = false
+        end
+      end
+
+      after do
+        Dynamoid.configure do |config|
+          config.create_table_on_save = true
+        end
+      end
+
+      it 'does not create table' do
+        model = klass.new
+
+        expect(klass).not_to receive(:create_table)
+
+        model.save
+      end
+    end
+
```

Comment:
I suppose we need this test for each method that persists a model, e.g. `#update_attributes` etc.

Reply:
Also we are not sure that testing `#update_attributes` and co provides that much value because they both call save. What do you think?

Reply:
It's OK to leave it as is.

I will add specs on my own to be sure behaviour isn't changed in case of some future refactoring.